src/Port.js

```diff
@@ -0,0 +1,24 @@
+class Port {
+    constructor(name){
+        this.name = name;
+        this.dockedShips = [];
+    };
+
+    addShip(ship) {
+        this.dockedShips.push(ship);
+    };
+
+    removeShip(ship) {
```

Comment:
RemoveShip and removeAnyship here is a little bit over complicated, even though it works perfectly, you could just use the built in array filter method, in the removeShip method, to remove the ship, and the you wouldn't have to use removeAnyShip at all :)
`````
    this.dockedShips = this.dockedShips.filter(
      (dockedShip) => dockedShip !== ship
    );
`````